src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationContext.cs

```diff
@@ -81,6 +81,22 @@ internal bool IsReplaying
             }
         }
 
+        internal TimeSpan LongRunningTimerIntervalDuration
+        {
+            get
+            {
+                return this.durabilityProvider.LongRunningTimerIntervalLength;
+            }
+        }
+
+        internal TimeSpan MaximumShortTimerDuration
+        {
+            get
+            {
+                return this.durabilityProvider.MaximumDelayTime;
+            }
+        }
+

```

Comment:
I realize these names were originally my suggestion, but now I'm thinking it would be best to call them the same as the property they're accessing inside `durablityProvider`.

What do you think of renaming these methods to be `LongRunningTimerIntervalLength` and `MaximumDelayTime`?

Reply:
I don't mind changing `LongRunningTimerIntervalDuration` to `LongRunningTimerIntervalLength`, since both of those names are confusing to me anyway 😛 But I would vote to keep not change `MaximumShortTimerDuration` to `MaximumDelayTime` because the latter feels more ambiguous to me, but I don't have strong opinions either way.

Also, should those naming changes be reflected in the SDK payload (and the SDK variable names themselves) too?

Reply:
I'm ok with that naming compromise - feel free to change the `LongRuning...` variable but keep the other one the same.

I don't think the SDK payload needs to have the same names, and the SDK can have whichever variable names it prefers so long as it parses the payload correctly :)

Reply:
Done :)